src/lib/cli.ts

```diff
@@ -8,10 +8,11 @@ import { MoxiedocSettings, process as moxieDocProcess } from './moxiedoc';
 process.argv[1] = 'moxiedoc';
 
 program
-  .version('0.2.0')
+  .version('0.2.2')
   .usage('[options] <dir ...>')
   .option('-o, --out <path>', 'output path, default: out')
-  .option('-t, --template <template>', 'template name')
+  .option('-t, --template <template>', 'template name, default: cli')
+  .option('-s, --structure <type>', 'output file structure, default: flat')
```

Comment:
This doesn't make sense, as the rest of the existing templates do not output a `flat` structure... I don't think we can specify the default here as it will vary based on the template default.